### DISCUSSION TITLE
Dockge seen on Noted 🤯

### DISCUSSION THREAD

**Discussion by Zaid-maker** [louislam/dockge#188] [General]
I am just using Google and saw dockge on news page on https://noted.lol/dockge/

I am really Proud of your work Louis! Amazing keep up the good work!

![Screenshot_20231126_154915](https://github.com/louislam/dockge/assets/53424436/7e57e345-92d8-4326-a300-4799bf19450e)

**Comment by louislam** [comment]
Great!
